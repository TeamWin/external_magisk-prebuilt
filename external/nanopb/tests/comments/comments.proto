syntax = "proto2";

// Message1Comment
message Message1
{
    // LeadingMemberComment
    required int32 member1 = 1;
    required int32 member2 = 2; // TrailingMemberComment
}

// Enum1Comment
enum Enum1
{
    // LeadingEnumComment
    ENUMVAL1 = 1;
    ENUMVAL2 = 2; // TrailingEnumComment
}
