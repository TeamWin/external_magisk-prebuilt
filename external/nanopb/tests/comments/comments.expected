Enum1Comment
LeadingEnumComment
ENUMVAL2.*TrailingEnumComment
Message1Comment
member2.*TrailingMemberComment

